tailormap-components/projects/core/src/lib/feature-form/form/form.component.ts

```diff
@@ -105,16 +110,26 @@ export class FormComponent implements OnDestroy, OnChanges, OnInit {
     this.formDirty = result;
   }
 
+  public setFormEditting(editting){
+    this.store$.dispatch(FormActions.setFormEditting({editting: editting}));
+  }
+
   public newItem(evt) {
     const type = LayerUtils.sanitizeLayername(evt.srcElement.id);
     this.store$.select(selectFormConfigForFeatureType, type)
       .pipe(takeUntil(this.destroyed))
       .subscribe(formConfig => {
-        this.actions.newItem$(this.features, type, formConfig).pipe(takeUntil(this.destroyed)).subscribe(features => {
-          this.features = features.features;
-          this.feature = features.feature;
-          this.initForm();
+        const objecttype = FormHelpers.capitalize(type);
+        const newFeature = this.featureInitializerService.create(objecttype, {
+          id: null,
+          clazz: type,
+          isRelated: true,
+          objecttype,
+          children: null,
         });
+        newFeature[formConfig.treeNodeColumn] = 'Nieuwe ' + formConfig.name;
```

Comment:
In Typescript you can now do this: 

```
const newFeature = this.featureInitializerService.create(objecttype, {
      id: null,
      clazz: type,
      isRelated: true,
      objecttype,
      children: null,
      [formConfig.treeNodeColumn]: 'Nieuwe ' + formConfig.name,
    });
```

And even better, replace the `'Nieuwe ' + formConfig.name` by \`Nieuwe ${formConfig.name}\`